src/topics/attestations-collection/attestations-collection.ts

```diff
@@ -1,16 +1,19 @@
 import { Badge } from "../badge";
 import { Group } from "../group";
 
-export type AttestationsCollectionsType = {
+export type AttestationsCollectionsConstructor = {
   groupFetcher: () => Promise<Group[]>;
   badge: Badge;
 };
 
+/**
+ * @description The representation of an AttestationsCollection that belongs to a specific Attester.
+ */
 export class AttestationsCollection {
   public groupFetcher: () => Promise<Group[]>;
   public badge: Badge;
 
-  constructor({ groupFetcher, badge }: AttestationsCollectionsType) {
+  constructor({ groupFetcher, badge }: AttestationsCollectionsConstructor) {
```

Comment:
Here we have renamed all the project for AttestationsCollectionsType from AttestationsCollectionsConstructor so we don't want to rename it back

Reply:
Updated, 

There are types outside this PR scope that have not been updated:
- GenerationContextConstructor
- GroupGeneratorConstructor